examples/mnist_net2net.py

```diff
@@ -193,12 +198,12 @@ def deeper2net_conv2d(teacher_w):
 
     # Arguments
         teacher_w: `weight` of previous conv2d layer,
-          of shape (filters, num_channel, kh, kw)
+          of shape (kw, kh, num_channel, filters)
```

Comment:
Shape is actually NHWC, i.e. height before width

Reply:
Fixed. It's filters, not images, so HWCK, right?

Reply:
Yes, that's correct